dask/dataframe/io/parquet/core.py

```diff
@@ -162,6 +162,8 @@ def read_parquet(
         data written by dask/fastparquet, not otherwise.
     storage_options : dict, default None
         Key/value pairs to be passed on to the file-system backend, if any.
+    open_options : dict, default None
+        Key/value pairs to be passed to ``dd.parquet.utils.open_parquet_file``.
```

Comment:
@martindurant - I'd like to get your thoughts on this:  After some thinking and experimenting, I felt that it was best to move the parquet-file opening logic into dedicated `open_parquet_file` utility.  The arguments to this utility are now exposed in `read_parquet` as a `open_options=` parameter.

@jorisvandenbossche - `open_parquet_file` has an `open_cb` key-word argument, allowing the files to be opened with a native pyarrow filesystem (e.g. `S3FileSystem.open_input_file`) if desired. In my experiments, the performance of using a native-file is certainly better than using `fs.open`, but is usually comparable to using `fsspec.parquet` (or slightly slower).  If/when optimizations in pyarrow start to outperform `fsspec.parquet` in the future, I will be happy to work out a way to make the native-file approach the default (altough, I expect translating `storage_options` may be a bit of a pain).